### PR TITLE
Language updates

### PR DESCRIPTION
**Pull request by filipviz**

### CODE REVIEW

File: test/E2E/Pay_Mint_Redeem_E2E.t.sol [hunk 1]
```diff
@@ -265,11 +265,13 @@ contract Test_TiersHook_E2E is TestBaseWorkflow {
         assertEq(IJB721TiersHook(dataHook).payCreditsOf(beneficiary), valueSent);
     }
 
-    // TODO: This needs care (fuzz fails with insuf reserve for val=10)
```
Comment (simon-something) [inline test/E2E/Pay_Mint_Redeem_E2E.t.sol]: tier prices start are tier*10, 9 should fails but 10 passed for me (hence the boundaries)

File: test/unit/adjustTier_Unit.t.sol [hunk 2]
```diff
@@ -1096,87 +1096,6 @@ contract Test_adjustTier_Unit is UnitTestSetup {
         hook.adjustTiers(tierConfigsToAdd, new uint256[](0));
     }
 
-    // TODO: I don't think this actually tests "locked" tiers.
```
Comment (simon-something) [inline test/unit/adjustTier_Unit.t.sol]: iic, we remove that functionality altogether

File: test/unit/redeem_Unit.t.sol [hunk 3]
```diff
@@ -201,7 +201,7 @@ contract Test_redeem_Unit is UnitTestSetup {
                 currency: uint32(uint160(JBConstants.NATIVE_TOKEN))
             }),
             useTotalSurplus: true,
-            redemptionRate: REDEMPTION_RATE,
+            redemptionRate: JBConstants.MAX_REDEMPTION_RATE,
```
Comment (simon-something) [inline test/unit/redeem_Unit.t.sol:204]: this was the only test using `REDEMPTION_RATE`, and it was assuming it was max ':0

File: test/unit/adjustTier_Unit.t.sol [hunk 4]
```diff
@@ -1193,12 +1193,83 @@ contract Test_adjustTier_Unit is UnitTestSetup {
                 allowOwnerMint: false,
                 useReserveBeneficiaryAsDefault: false,
                 transfersPausable: false,
-                useVotingUnits: false
+                useVotingUnits: false  // <-- If false, voting power is based on tier price
             });
         }
 
-        // TODO: Shouldn't this be `.votingUnits` (and not `.category`)?
```
Comment (simon-something) [inline test/unit/adjustTier_Unit.t.sol]: useVotingUnits being false, the voting power is established with the tier price -> this setup is not needed here + added a test when using the voting unit

File: test/unit/mintFor_mintReservesFor_Unit.t.sol [hunk 5]
```diff
@@ -229,31 +229,6 @@ contract Test_mintFor_mintReservesFor_Unit is UnitTestSetup {
         }
     }
 
-    function test_useDefaultReservedBeneficiary() public {
```
Comment (simon-something) [inline test/unit/mintFor_mintReservesFor_Unit.t.sol]: this is now tested in test_mintPendingReservesFor_mintMultipleReservedTokens

File: script/LaunchProjectFor.s.sol [hunk 6]
```diff
@@ -1,43 +1,42 @@
 /**
  * DEPRECATED - Use jbm for now
```
Comment (simon-something) [inline script/LaunchProjectFor.s.sol:2]: Not sure what we should do with this (this is a deploy script, not a test)
@mejango did u use this recently? Maybe we'll need it for some test deployment on Sepolia (if any?)

Comment (mejango) [reply script/LaunchProjectFor.s.sol:2]: delete

**Pull request merged**